src/olympia/amo/views.py

```diff
@@ -55,6 +55,22 @@ def monitor(request):
     return http.HttpResponse(json.dumps(status_summary), status=status_code)
 
 
+@never_cache
+@non_atomic_requests
+def client_info(request):
+    if getattr(settings, 'ENV', None) != 'dev':
```

Comment:
having a setting that defaulted to `True` in the dev settings file but was settable via env files would give us more flexibility if it needed to be tested on stage, etc, too but if it's a short term thing then this works.